### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by laumcaicedo-zz**
adding cleaning paragraph

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: labs/01_ingestion_with_glue/README.md [hunk 1]
```diff
@@ -111,6 +111,9 @@ NOTE: “AWSGlueServiceRole” is an AWS Managed Policy to provide Glue with nee
 In AWS Glue can create an environment—known as a development endpoint—that you can use to iteratively develop and test your extract, transform, and load (ETL) scripts. 
 You can then create a notebook that connects to the endpoint, and use your notebook to author and test your ETL script. When you're satisfied with the results of your development process, you can create an ETL job that runs your script. With this process, you can add functions and debug your scripts in an interactive manner.
 
+**NOTE: Please be aware that AWS Glue allocates 5 DPUs to each development endpoint by default, you are billed $0.44 per DPU-Hour, therefore PLEASE make sure to clean/ delete this endpoint once you have finish all the labs - follow the instructions in [lab - Wrap Up and Clean](https://github.com/laumcaicedo/bring-your-own-data-labs/tree/master/labs/99_Wrap_up_and_clean)**
```
Comment (BastLeblanc) [inline labs/01_ingestion_with_glue/README.md]: The link is wrong (pointing to your github account) can you make it relative ?

**Review by BastLeblanc — COMMENTED** [review]
see comments inline, one broken link to fix

File: labs/01_ingestion_with_glue/README.md (hunk 1 above)
Comment (laumcaicedo-zz) [reply labs/01_ingestion_with_glue/README.md]: Link has been updated

**Review by BastLeblanc — APPROVED** [review]
approved, you can merge !

**Pull request merged**